invenio_assets/webpack.py

```diff
@@ -10,12 +10,104 @@
 
 from __future__ import absolute_import, print_function
 
-from flask_webpackext import WebpackBundleProject
-from pywebpack import bundles_from_entry_point
+import os
+from collections import OrderedDict
+
+from flask import current_app, request
+from flask_webpackext import WebpackBundle, WebpackBundleProject
+from flask_webpackext.manifest import JinjaManifest, JinjaManifestLoader
+from markupsafe import Markup
+from pywebpack import ManifestEntry, UnsupportedExtensionError, \
+    bundles_from_entry_point
 
 project = WebpackBundleProject(
     __name__,
     project_folder='assets',
     config_path='build/config.json',
     bundles=bundles_from_entry_point('invenio_assets.webpack'),
 )
+
+
+class WebpackThemeBundle(object):
+    """Webpack themed bundle."""
+
+    def __init__(self, import_name, folder, themes=None):
+        """Initialize webpack bundle.
+
+        :param path: Absolute path to the folder where the assets are
+            located.
+        :param themes: Dictionary of theme -> .
+
+        """
+        self.themes = {}
+        for theme, bundle in themes.items():
+            self.themes[theme] = WebpackBundle(
+                import_name,
+                os.path.join(folder, theme),
+                **bundle
+            )
+        self._dummy_bundle = WebpackBundle(import_name, folder)
+
+    @property
+    def _active_theme_bundle(self):
+        for theme in current_app.config['APP_THEME']:
```

Comment:
What if ``APP_THEME`` is undefined? Do we throw an error, or continue silently?

```suggestion
        for theme in current_app.config.get('APP_THEME', []):
```

Reply:
I've added a `default` parameter which, specifies the `bootstrap3` bundle... This gets used only in case there's no `APP_THEME` set, to make it backward compatible.